packages/data/src/onboarding/selectors.ts

```diff
@@ -21,9 +22,9 @@ export const getTasksStatus = (
 	return state.tasksStatus || {};
 };
 
-const initialTaskLists: TaskList[] = [];
+const initialTaskLists: TaskListType[] = [];
 
-export const getTaskLists = ( state: OnboardingState ): TaskList[] => {
+export const getTaskLists = ( state: OnboardingState ): TaskListType[] => {
 	return state.taskLists || initialTaskLists;
 };
 
```

Comment:
Github won't let me comment on unchanged lines, but on line 69 I believe `TaskList[]` needs to be `TaskListType[]`